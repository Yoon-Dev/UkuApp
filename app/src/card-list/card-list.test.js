import React from 'react';
import ReactDOM from 'react-dom';
import { render, unmountComponentAtNode } from "react-dom";
import { act } from "react-dom/test-utils";
import CardList from './card-list';
import pretty from "pretty";


let container = null;

beforeEach( () => {
  container = document.createElement("div");
  document.body.appendChild(container);
});

afterEach( () => {
  unmountComponentAtNode(container);
  container.remove();
});

it('Load 2 card element, the title of the first card equals Jingle Bells', async () => {

act(() => {
    render(<CardList></CardList>, container);
});
const carlist = document.querySelector("[data-testid='cardlist]");
expect(
  pretty(container.innerHTML)
).toMatchSnapshot();


})





// }); 
