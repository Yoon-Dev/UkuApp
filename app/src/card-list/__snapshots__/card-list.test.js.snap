// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Load 2 card element, the title of the first card equals Jingle Bells 1`] = `"<div class=\\"card-list\\" data-testid=\\"cardlist\\" data-testnbr=\\"0\\">Wait...</div>"`;
